tabs/src/components/ui/OverallCode.jsx

```diff
@@ -0,0 +1,146 @@
+export const NestingThemesCode = 
```

Comment:
Use string in code will make it hard to maintain, better to move it to text file, then using webpack or other bundle tool help you to import these strings to your code.